src/transformers/models/maskformer/modeling_maskformer.py

```diff
@@ -1977,19 +1977,19 @@ def __init__(self, in_features: int, out_features: int, kernel_size: int = 3, pa
                 The number of outputs features (channels).
         """
         super().__init__()
-        self.layers = [
+        self.layers = nn.Sequential(
             nn.Conv2d(in_features, out_features, kernel_size=kernel_size, padding=padding, bias=False),
             nn.GroupNorm(32, out_features),
             nn.ReLU(inplace=True),
-        ]
-        for i, layer in enumerate(self.layers):
-            # Provide backwards compatibility from when the class inherited from nn.Sequential
-            # In nn.Sequential subclasses, the name given to the layer is its index in the sequence.
-            # In nn.Module subclasses they derived from the instance attribute they are assigned to e.g.
-            # self.my_layer_name = Layer()
-            # We can't give instance attributes integer names i.e. self.0 is not permitted and so need to register
-            # explicitly
-            self.add_module(str(i), layer)
+        )
+        # for i, layer in enumerate(self.layers):
+        #     # Provide backwards compatibility from when the class inherited from nn.Sequential
+        #     # In nn.Sequential subclasses, the name given to the layer is its index in the sequence.
+        #     # In nn.Module subclasses they derived from the instance attribute they are assigned to e.g.
+        #     # self.my_layer_name = Layer()
+        #     # We can't give instance attributes integer names i.e. self.0 is not permitted and so need to register
+        #     # explicitly
+        #     self.add_module(str(i), layer)
```

Comment:
will remove before merge

Reply:
Could be that @amyeroberts added this, since she opened a PR to remove inheritance from `nn.Sequential`

Reply:
Right, I think I added this, but it's actually fine to have `nn.Sequential` within the `__init__`

Reply:
This will break backward compatibility with all existing checkpoints as you are renaming weights which will be incompatible with the existing checkpoints. It can't be commented out or removed.